### PR TITLE
Removed and replaced dsql in tutorials as it is now obsolete 

### PR DESCRIPTION
**Pull request by hevansDev**
### Description

These 2 tutorials used dsql, which is a dead project, no longer supported, and doesn’t even work for Python3.

I have removed reference to dsql and replace it with instructions on how to make an API call instead.

This PR has:

- [x] been self-reviewed.

### CODE REVIEW

File: docs/tutorials/tutorial-transform-spec.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+## Sample data
+
+We've included sample data for this tutorial at `quickstart/tutorial/transform-data.json`, reproduced here for convenience:
+
+```json
+{"timestamp":"2018-01-01T07:01:35Z","animal":"octopus",  "location":1, "number":100}
+{"timestamp":"2018-01-01T05:01:35Z","animal":"mongoose", "location":2,"number":200}
+{"timestamp":"2018-01-01T06:01:35Z","animal":"snake", "location":3, "number":300}
+{"timestamp":"2018-01-01T01:01:35Z","animal":"lion", "location":4, "number":300}
```
Comment (asdf2014) [inline docs/tutorials/tutorial-transform-spec.md:42]:
```suggestion
{"timestamp": "2018-01-01T07:01:35Z","animal": "octopus","location": 1,"number": 100}
{"timestamp": "2018-01-01T05:01:35Z","animal": "mongoose","location": 2,"number": 200}
{"timestamp": "2018-01-01T06:01:35Z","animal": "snake","location": 3,"number": 300}
{"timestamp": "2018-01-01T01:01:35Z","animal": "lion","location": 4,"number": 300}
```

File: docs/tutorials/tutorial-transform-spec.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+  ~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  ~ KIND, either express or implied.  See the License for the
+  ~ specific language governing permissions and limitations
+  ~ under the License.
+  -->
+
+
+This tutorial will demonstrate how to use transform specs to filter and transform input data during ingestion.
+
+For this tutorial, we'll assume you've already downloaded Apache Druid as described in
```
[the remark this answers is not in the document]
Comment (hevansDev) [reply docs/tutorials/tutorial-transform-spec.md:29]: Resolved by changes made by @edgar2020 https://github.com/apache/druid/pull/16845

File: docs/tutorials/tutorial-transform-spec.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+the [single-machine quickstart](index.md) and have it running on your local machine.
+
+It will also be helpful to have finished [Load a file](../tutorials/tutorial-batch.md) and [Query data](../tutorials/tutorial-query.md) tutorials.
+
+## Sample data
+
+We've included sample data for this tutorial at `quickstart/tutorial/transform-data.json`, reproduced here for convenience:
```
[the remark this answers is not in the document]
Comment (hevansDev) [reply docs/tutorials/tutorial-transform-spec.md:36]: Resolved by changes made by @edgar2020 https://github.com/apache/druid/pull/16845

File: docs/tutorials/tutorial-transform-spec.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+```
+
+## Query the transformed data
+
+Let's issue a `select * from "transform-tutorial";` query to the SQL API and see what data was ingested.
+
+```bash
+curl -X POST -H "Content-Type: application/json" -d '{"query": "select * from \"transform-tutorial\""}' http://localhost:8888/druid/v2/sql
+```
+
```
[the remark this answers is not in the document]
Comment (hevansDev) [reply docs/tutorials/tutorial-transform-spec.md:147]: Resolved by changes made by @edgar2020 https://github.com/apache/druid/pull/16845

**Pull request closed without merging**